### PR TITLE
Unified customization available for unread message divider

### PR DESCRIPTION
**Pull request by yurii-glia**
MOB-1961/0

### CODE REVIEW

File: GliaWidgets/Sources/View/Chat/ChatStyle.swift [hunk 1]
```diff
@@ -118,51 +118,45 @@ public class ChatStyle: EngagementStyle {
...
         )
         unreadMessageIndicator.apply(
-            configuration: configuration?.unreadIndicator,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.unreadIndicator, assetsBuilder: assetsBuilder
         )
         operatorTypingIndicator.apply(configuration: configuration?.typingIndicator)
 
+        unreadMessageDivider.apply(
+            lineColor: configuration?.newMessagesDividerColor, text: configuration?.newMessagesDividerText,
```
Comment (gersonnoboa) [inline GliaWidgets/Sources/View/Chat/ChatStyle.swift:156]: I guess the `text` can also be on its own line

File: GliaWidgets/Sources/View/Chat/ChatStyle.swift
Comment (gersonnoboa) [inline GliaWidgets/Sources/View/Chat/ChatStyle.swift:1]: Any reason for moving all the parameters to one line instead of two as they were before?

**Review by igorkravchenko — APPROVED** [review]

File: GliaWidgets/Sources/View/Chat/ChatStyle.swift (hunk 1 above)
Comment (yurii-glia) [reply GliaWidgets/Sources/View/Chat/ChatStyle.swift:156]: I did it because of SwiftLint warning

Comment (yurii-glia) [reply GliaWidgets/Sources/View/Chat/ChatStyle.swift]: fixed

File: GliaWidgets/Sources/View/Chat/ChatStyle.swift
Comment (yurii-glia) [reply GliaWidgets/Sources/View/Chat/ChatStyle.swift:1]: fixed

**Review by gersonnoboa — APPROVED** [review]
I will approve but there's a SwiftLint warning still.

**Pull request merged**